src/js/page_sections/Header.js

```diff
@@ -73,11 +75,15 @@ class Header extends Component {
             </div>
           </div>
         </div>
-        <Menu isMenuOpen={this.state.menuIsOpen} toggleMenu={this.toggleMenu} />
+        <Menu
+          isMenuOpen={this.state.menuIsOpen}
+          toggleMenu={this.toggleMenu}
+          navigation={this.props.navigation[this.props.intl.locale]}
```

Comment:
I think props is already destructured on line 44 -- you could add navigation to that destructure statement and shorten line 81 to `navigation={navigation[intl.locale]}`